index.html

```diff
@@ -1,12 +1,337 @@
+            </div>
+        </div>
+        <div class="container-1">
+            <div class="items">
+                <h1>Uber Navigation</h1>
+                <ul>
+                    <li>UBER</li>
+                    <li>Lead Deveveloper</li>
+                    <li>2018</li>
+                </ul>
+                <p>
+                    A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive
+                    computer:
+                    your car.
+                </p>
+                <ul class="btn-div">
+                    <li><a href="" class="btn">HTML</a></li>
+                    <li><a href="" class="btn">CSS</a></li>
+                    <li><a href="" class="btn ruby">Ruby on rails</a></li>
+                    <li><a href="" class="btn">javascript</a></li>
+                </ul>
+                <button class="last-btn" type="button">See Project</button>
+            </div>
+            <img src="Media/Snapshoot Portfolio (1).svg" />
+        </div>
+    </section>
+    <section class="section-3">
+        <div class="container-3">
+            <div class="about-div">
+                <h1 class="about">About Myself</h1>
+                <p>
+                    Hello I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.
+                </p>
+                <p class="social-heading">LET’S CONNECT</p>
+                <ul class="socials">
+                    <li>
+                        <a href="#"><img src="Media/twitter.svg" /> </a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="Media/Linkedin icon.svg" /></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="Media/mediu.svg" /></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="Media/github.svg" /></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="Media/angellist.svg" /></a>
+                    </li>
+                </ul>
+                <button class="last-btn m-b" type="button">Get My Resume</button>
+            </div>
+            <div class="languages">
+                <ul class="languages-div language">
+                    <li>Languages</li>
+                    <i class="fa fa-angle-down" aria-hidden="true"></i>
+                </ul>
+                <div class="wrapper">
+                    <div class="languages-div skills">
+                        <img src="Media/Ellipse 1.png" />
+                        <h5 class="m-l">javascript</h5>
+                    </div>
+                    <div class="languages-div skills">
+                        <img src="Media/Ellipse 1 (1).png" />
```

Comment:
[Optional] but recommended for professional purposes.
I would recommend renaming your images and assets to use small caps and remove spaces between the image names for SEO purposes as well semantics.